src/components/Footer/Links.jsx

```diff
@@ -21,6 +21,12 @@ export default () => (
       >
         <IconSteam style={{ width: 14, height: 14 }} />
       </IconButton>
+      {location.location === '/' &&
```

Comment:
Alternatively, you could put the attribution with the image like we currently do for the carry page:
https://www.opendota.com/carry